src/harness/testcases/grant_testcase.py

```diff
@@ -478,3 +478,140 @@ def test_WINFF_FT_S_GRA_23(self):
       self.assertTrue('grantId' in resp)
       self._assert_valid_response_format_for_approved_grant(resp)
       self.assertEqual(resp['response']['responseCode'], 0)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_GRA_26(self):
+    """Two grant requests. #1 Successful, #2 Unsuccessful.
+
+    Returns 0 (NO_ERROR) for successful and 103 (INVALID_VALUE) for unsuccessful
+    """
+    # Register two devices
+    device_a = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
```

Comment:
Minor comment- it seems we are using four spaces for indent (except in the function definition). Make this four spaces?